### PR TITLE
RC: New Packaging Documentation updates

### PR DESCRIPTION
**Pull request by cmilesb**
Base staging link: https://docs.redis.com/staging/rc-packaging/rc/
Databases: https://docs.redis.com/staging/rc-packaging/rc/databases/
Subscriptions: https://docs.redis.com/staging/rc-packaging/rc/subscriptions/

### CODE REVIEW

File: content/rc/cloud-integrations/aws-marketplace/aws-bedrock/set-up-redis.md [hunk 1]
```diff
@@ -48,15 +48,19 @@ To set up a Redis Cloud instance for Bedrock, you need to:
...
+### [Create a database]({{<relref "/rc/databases/create-database/create-pro-database-new">}}) {#create-sub} 
 
-1. In the [admin console](https://app.redislabs.com/), select **Add new subscription**. 
+1. In the [admin console](https://app.redislabs.com/), select **New database**. 
 
-    {{<image filename="images/rc/button-subscription-new.png" width="50%" alt="The New subscriptions button in the admin console menu." >}}{{< /image >}}
+    {{<image filename="images/rc/button-database-new.png" alt="The New Database button creates a new database." width="120px">}}{{< /image >}}
 
-1. When the **New subscription** page appears, select **Flexible plans**.
+1. When the **New database** page appears, select **Vector database** from the use case panel.
```
Comment (cmilesb) [inline content/rc/cloud-integrations/aws-marketplace/aws-bedrock/set-up-redis.md:57]:
```suggestion
1. When the **New database** page appears, select **Vector search** from the use case panel.
```

File: content/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
 weight: $weight
 alwaysopen: false
 categories: ["RC"]
+aliases: [ /rc/subscriptions/create-fixed-subscription/fixed-plan-details,
+            /rc/subscriptions/view-fixed-subscription/fixed-plan-details ]
+
 ---
 
-## Current Fixed plans
+Redis Cloud Essentials is cost-efficient and designed for low-throughput scenarios. It support a range of availability, persistence, and backup options.
```
Comment (dwdougherty) [inline content/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md:12]:
```suggestion
Redis Cloud Essentials is cost-efficient and designed for low-throughput scenarios. It supports a range of availability, persistence, and backup options.  
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Just one nit after a speedy skim.

**Review by mich-elle-luna — APPROVED** [review]
thank you!

**Pull request merged**